routing/routing_tests/index_graph_tools.hpp

```diff
@@ -87,6 +90,21 @@ class TestIndexGraphLoader final : public IndexGraphLoader
   unordered_map<NumMwmId, unique_ptr<IndexGraph>> m_graphs;
 };
 
+class TestTransitGraphLoader : public TransitGraphLoader
+{
+public:
+  // TransitGraphLoader overrides:
+  ~TestTransitGraphLoader() override = default;
+
+  TransitGraph & GetTransitGraph(NumMwmId mwmId, IndexGraph & indexGraph) override;
+  virtual void Clear() override;
```

Comment:
No need in "virtual" and "override" in the same time.

Reply:
Please, leave just "override".

Reply:
done